backend/database/startup.docker.py

```diff
@@ -0,0 +1,123 @@
+import logging
```

Comment:
Why do we have to do a full duplicate and not link to other existing files for these features?

Reply:
I know it is far from ideal as this would imply double maintenance if anything changes. It was the fastest way I had to recreate the environment without affecting those who develop without docker (everyone so far).
My idea is to work this by replacing the 'localhosts' and other aspects with environment variables. But that would require to change more stuff. 
I can continue working on my fork and later try to integrate once I have the env variables done.